### ISSUE TITLE
Issue with reading this xlsx file

### ISSUE THREAD

**Issue by Lambik** [PHPOffice/PhpSpreadsheet#2067]
This is:

```
- [x] a bug report
```

### What is the expected behavior?

The xlsx file can be opened and used

### What is the current behavior?

Loading the file yields the error `Unable to identify a reader for this file`

### What are the steps to reproduce?

The file specified is attached to the issue: [TRAINING.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/6435157/TRAINING.xlsx)

```php
<?php

use PhpOffice\PhpSpreadsheet\IOFactory;

require __DIR__ . '/vendor/autoload.php';

$excel = IOFactory::load('TRAINING.xlsx');
```

### Which versions of PhpSpreadsheet and PHP are affected?

I'm on PHP 7.4 with PhpSpreadsheet 1.17.1 (latest at the time of the issue).

### Cause

I've investigated myself, and the issue is that the xlsx file seems to use a different xml namespace than expected. It's all based on `http://purl.oclc.org/ooxml/officeDocument/relationships` instead of the usual `http://schemas.openxmlformats.org/officeDocument/2006/relationships`. This makes the function `getWorkbookBaseName` and the subsequent `load` fail.

I don't know how the author of the file created the file, they haven't told me yet (if/when I know, I'll update the issue).

**Comment by oleibman** [comment]
Ouch. I am just starting to look at namespace prefixes, an effort which is not quite ready yet. However, my work on that area depends on having known *constant* namespaces, and your sample file (thank you for making it available) still won't work for that. I see that you have opened PR #2068. I will study it to see if I can integrate your changes and mine.

**Comment by Lambik** [comment]
Glad to have helped, unfortunately I don't know if my PR covers all advanced cases in the new namespace (since I can't create a new file myself). All I know is that my changes work on the linked file, which is a good start..

**Comment by oleibman** [comment]
Agreed about "good start". My changes are already so complex that they will almost certainly happen incrementally in several stages, so I too will have "advanced cases which might not be covered". For now, I have placed your PR in draft status. If it looks like my work will take too long, I will return yours to ready status.

**Comment by oleibman** [comment]
@Lambik - have you been able to run tests against master to confirm that I have implemented your changes correctly?

**Comment by PowerKiKi** [comment]
Should be solved in #2173
